### ISSUE TITLE
Errors in OpenFAST tests

### ISSUE THREAD

**Issue by jrood-nrel** [Exawind/nalu-wind#693]
Seems to be missing an input file:
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  FAST_InitializeAll:FAST_Init:FAST_ReadPrimaryFile:Invalid numerical input for file "../nrel5MWactuatorLine/nrel5mw.fst" occurred while trying to read NLinTimes.
```

**Comment by psakievich** [comment]
@jrood-nrel This was a copy and paste error that snuck in.  I accidentally copied one parameter so it got included twice.  Should be fixed in #692, but let's confirm with the runs tonight.

**Comment by psakievich** [comment]
Looks like we've got some pretty big diffs across all tests except `nrel5MWactuatorLine` which I just reblessed for the GPU tests.  I believe these are due to the changes in OpenFAST since basically nothing has changed in this code since the Q2 milestone.  I'm wondering if we should make at least one nightly test build a fresh version of OpenFAST each night so we catch these changes and can provide better feedback to the OpenFAST team if we see a large diff from a change?
